Modules/Stage/Services/StageDeleteService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Modules\Stage\Services;
+
+use Modules\Stage\Models\Stage;
+use Modules\Stage\Repositories\Interfaces\StageRepositoryInterface;
+
+/**
+ * Class StageService
+ * @package Modules\Stage\Services
+ */
+class StageDeleteService {
+    /**
+     * @var StageRepositoryInterface
+     */
+    private $stageRepo;
+
+    /**
+     * StageGetService constructor.
+     * @param StageRepositoryInterface $stageRepo
+     */
+    public function __construct(StageRepositoryInterface $stageRepo) {
+        $this->stageRepo = $stageRepo;
+    }
+
+    /**
+     * @param array $data
+     * @return Stage|null
+     */
+    public function delete($stage_id) : ?bool {
+        return $this->stageRepo->deleteWhere("stage_id","=",$stage_id);
```

Comment:
return $this->stageRepo->delete($id);

(por default las llaves primarias se deben de llamar `id` en laravel)